packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -114,7 +118,7 @@ export interface PrismaClientOptions {
       allowTriggerPanic?: boolean
     }
   }
-}
+} & ({} | {})
```

Comment:
Do you think it's worth to use an Either type for this one?

Reply:
Accidental leftover from attempt to make the properties mutually exclusive. I managed, but TS error message in case both are used was very misleading, so I've decided it will be more helpful if we do runtime check only.